include/abb_librws/rws_client.h

```diff
@@ -512,6 +512,32 @@ class RWSClient : public POCOClient
    */
   RWSResult logout();
 
+  /**
+   * \brief A method for registering a user as local.
+   *
+   * \param username specifying the user name.
+   * \param application specifying the external application.
+   * \param location specifying the location.
+   *
+   * \return RWSResult containing the result.
+   */
+  RWSResult registerLocalUser(std::string username = "Default User",
+                              std::string application = "ExternalApplication",
+                              std::string location = "ExternalLocation");
```

Comment:
Should all of these be constants somewhere, instead of provided as default arguments?

Reply:
same for some of the default args used / introduced in other files.

Reply:
Yes, that is better 👍 I will change it